lib/exq/manager/server.ex

```diff
@@ -160,13 +156,25 @@ defmodule Exq.Manager.Server do
                    queues: opts[:queues],
                    pid: self(),
                    poll_timeout: opts[:poll_timeout],
-                   scheduler_poll_timeout: opts[:scheduler_poll_timeout]
+                   scheduler_poll_timeout: opts[:scheduler_poll_timeout],
+                   started_at: Time.unix_seconds,
                    }
 
     check_redis_connection(opts)
+
+    # Cleanup stale stats
+    rescue_timeout(fn ->
+      Exq.Stats.Server.cleanup_host_stats(state.stats, state.namespace, state.node_id, state.pid)
+    end)
+
+    HeartbeatServer.start_link(state)
```

Comment:
Starting a process this way is not recommended. The proper way of starting it is to add this to the supervision tree:  https://github.com/akira/exq/blob/master/lib/exq/support/mode.ex#L26